documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -6,26 +6,54 @@
 = Handling high volumes of messages
 
 [role="_abstract"]
-If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it.
+If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it. 
```

Comment:
I wonder if we should clarify what to consider a high volume of data? This could be a source cluster with average throughput but a lot of existing data.

Reply:
I wasn't sure if we needed this. But I guess an example or two might help.

I added:

> Operations like tracking web activity can generate a high volume of messages. 
> But even a source cluster with moderate throughput can produce a high volume of messages if it has a lot of existing data.